tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/SeleniumInfrastructureTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Extensibility;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+
+namespace Microsoft.Identity.Test.Integration.net45.SeleniumTests
+{
+    [TestClass]
+    public class SeleniumInfrastructureTests
+    {
+        private static readonly string[] s_scopes = new[] { "user.read" };
+
+        #region MSTest Hooks
+        /// <summary>
+        /// Initialized by MSTest (do not make private or readonly)
+        /// </summary>
+        public TestContext TestContext { get; set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            TestCommon.ResetInternalStaticCaches();
+        }
+
+        #endregion
+
+        [TestMethod]
+        public async Task FailingTest_SeleniumFailureAsync()
```

Comment:
Tests for test infrastructure :)

Reply:
Good idea!

---
In reply to: [287452257](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1172#discussion_r287452257) [](ancestors = 287452257)